torchgeo/trainers/classification.py

```diff
@@ -64,6 +64,13 @@ def config_model(self) -> None:
                 state_dict = get_weight(weights).get_state_dict(progress=True)
             self.model = utils.load_state_dict(self.model, state_dict)
 
+        # Freeze backbone and unfreeze classifier head
+        if self.hyperparams.get("freeze_backbone", False):
+            for param in self.model.parameters():
+                param.requires_grad = False
+            for param in self.model.get_classifier().parameters():
```

Comment:
What does `get_classifier` look like for various models? Is it always everything except the last layer?

Reply:
They added this because sometimes they name the head differently, e.g. head, fc, cls, etc.